Stripe/STPPaymentMethodsViewController.m

```diff
@@ -267,15 +288,14 @@ - (instancetype)initWithConfiguration:(STPPaymentConfiguration *)configuration
         self.navigationItem.title = STPLocalizedString(@"Loading…", @"Title for screen when data is still loading from the network.");
 
         WEAK(self);
-        [loadingPromise onSuccess:^(STPPaymentMethodTuple *tuple) {
-            STRONG(self);
-            self.paymentMethods = tuple.paymentMethods;
-            self.selectedPaymentMethod = tuple.selectedPaymentMethod;
```

Comment:
Looked like we were never reading these properties so just getting rid of this extra cached state.